### ISSUE TITLE
Cannot follow in-page links with VoiceOver in iOS 11

### ISSUE THREAD

**Issue by selfthinker** [alphagov/reported-bugs#36]
**Upstream bug:** https://bugs.webkit.org/show_bug.cgi?id=179011
**Date:** 2017-10-30
**Reported by:** Anika Henke
**Related to:** n/a

---

### Overview

In iOS 11 you can currently not follow any in-page links to anything but inputs.

### Steps to reproduce

1. Activate VoiceOver (on iOS 11)
2. Open http://jsbin.com/wumitiw in Safari
3. Navigate to any of the links (except "link to input")
4. Follow the link (i.e. double tap)

### Actual result

The focus moves to the correct element for a split second and then moves back onto the link that was clicked on.

### Expected result

The focus should go and stay where the link pointed to.

**Comment by selfthinker** [comment]
Axess Lab blogged about the very same issue. I sent them some comments which they published at the [bottom of that blog post](https://axesslab.com/skip-links/#update-3-a-comment-from-gov-uk).

**Comment by selfthinker** [comment]
I just re-checked this in iOS 12 and iOS 13.
The issue is still in iOS 12 but seems to have been fixed in iOS 13. 🎉

**Comment by querkmachine** [comment]
Unfortunately this bug seems to have reappeared in iOS 18 and persists into iOS 26. 😔 

A new bug report, using the same code example and reproduction steps as this issue, has been opened here: https://bugs.webkit.org/show_bug.cgi?id=295549

I'm going to re-open this issue and update the original post to remind us to keep an eye on it.
